### ISSUE TITLE
Support deploy keys

### ISSUE THREAD

**Issue by dotfloat** [equinor/komodo#184]
Allow for users to specify deploy keys for each package. The keys themselves are stored somewhere else, eg `/private/f_komodo/keys`

**Comment by markusdregi** [comment]
Good idea! The keys could even be kept as Jenkins secrets and persisted to a file only while doing the GitHub interrogation (using GIT_SSH_COMMAND or similar). I don't know which is better, but it leaves us at least with multiple options to consider for how to keep track of the keys...
